src/pages/task/types.ts

```diff
@@ -0,0 +1,26 @@
+export interface ValidInitialQueryParams {
```

Comment:
i feel like this file belongs i the root of pages because it relates to task page as a whole. putting in subdirectory of task page doesn't feel quite right

Reply:
But presumably there will be many more pages in the future - spruce won't just be the task page

Reply:
yes and that's fine with me. within a `types` dir we can have files for each page and then have all exported through and index file if we'd like

Reply:
Exporting all of our interfaces/types for each page through an index file may make it more difficult to come up with simple unique names for interfaces/types for all our pages.

Reply:
they don't have to be exported from an index. i still don't think the types belong in the page's subdirectoy tho